dispute.go

```diff
@@ -0,0 +1,65 @@
+package braintree
+
+import "time"
+
+const (
+
+	// Dispute kinds
+	DisputeKindChargeback     = "chargeback"
+	DisputeKindPreArbitration = "pre_arbitration"
+	DisputeKindRetrieval      = "retrieval"
+
+	// Dispute reasons
+	DisputeReasonCancelledRecurringTransaction = "cancelled_recurring_transaction"
+	DisputeReasonCreditNotProcessed            = "credit_not_processed"
+	DisputeReasonDuplicate                     = "duplicate"
+	DisputeReasonFraud                         = "fraud"
+	DisputeReasonGeneral                       = "general"
+	DisputeReasonInvalidAccount                = "invalid_account"
+	DisputeReasonNotRecognized                 = "not_recognized"
+	DisputeReasonProductNotReceived            = "product_not_received"
+	DisputeReasonProductUnsatisfactory         = "product_unsatisfactory"
+	DisputeReasonTransactionAmountDiffers      = "transaction_amount_differs"
+
+	// Dispute statuses
+	DisputeStatusAccepted = "accepted"
+	DisputeStatusDisputed = "disputed"
+	DisputeStatusExpired  = "expired"
+	DisputeStatusOpen     = "open"
+	DisputeStatusLost     = "lost"
+	DisputeStatusWon      = "won"
+)
+
+type Dispute struct {
+	XMLName         string             `xml:"dispute"`
+	AmountDisputed  *Decimal           `xml:"amount-disputed"`
+	AmountWon       *Decimal           `xml:"amount-won"`
+	CaseNumber      string             `xml:"case-number"`
+	CreatedAt       *time.Time         `xml:"created-at"`
+	CurrencyISOCode string             `xml:"currency-iso-code"`
+	Evidence        []*DisputeEvidence `xml:"evidence>evidence"`
+
```

Comment:
What's the motivation for splitting the two sets of fields?

Reply:
Don't get the question. Just specify how would you implement this.

Reply:
`Evidence` was declared like this: https://github.com/lionelbarrow/braintree-go/blob/master/subscription.go#L48

Reply:
I was meaning what's the reason for the empty line between the fields? It's fine though.

Reply:
@leighmcculloch ah, ok. That was done accidentally. Will fix.